src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,40 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+  }
 }
 
```

Comment:
I believe linter currently doesn't like this notation. You can move in outside Animal object, like this. There is no error here per se.
```suggestion

  static addAlive(animal) {
  Animal.alive.push(animal)
  }
  constructor(name, health = 100) {
    this.name = name;
    this.health = health;
  }
}

Animal.alive = [];
```

Reply:
it's due to node version we using which doesn't support static fields